packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -0,0 +1,57 @@
+import $ from '@js/core/renderer';
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+
+import Widget from '../widget';
+
+const CHAT_MESSAGE_AVATAR_CLASS = 'dx-chat-message-avatar';
+const CHAT_MESSAGE_AVATAR_LETTERS_CLASS = 'dx-chat-message-avatar-letters';
+
+export interface AvatarOptions extends WidgetOptions<Avatar> {
+  name?: string;
+}
+
+class Avatar extends Widget<AvatarOptions> {
+  _getDefaultOptions(): AvatarOptions {
+    return {
+      ...super._getDefaultOptions(),
+      name: '',
+    };
+  }
+
+  _getAvatarInitials(name: string): string {
+    const initials = name.charAt(0).toUpperCase();
+
+    return initials;
+  }
+
+  _initMarkup(): void {
+    $(this.element()).addClass(CHAT_MESSAGE_AVATAR_CLASS);
+
+    super._initMarkup();
+
+    const $letters = $('<div>').addClass(CHAT_MESSAGE_AVATAR_LETTERS_CLASS);
+
+    const { name } = this.option();
+
+    if (name) {
+      const text = this._getAvatarInitials(name);
+
+      $letters.text(text);
+    }
+
+    $letters.appendTo(this.element());
```

Comment:
What will the avatar look like if the name is an empty string—an empty circle? Will we consider something else?

Reply:
We don't have a requirement for that right now